build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,275 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var BBPromise = require('bluebird');
+var argv = require('minimist')(process.argv.slice(2));
+var assert = require('assert');
+var child_process = require('child_process');
+var config = require('../config');
+var extend = require('util')._extend;
+var fs = require('fs-extra');
+var git = require('gulp-git');
+var gulp = require('gulp-help')(require('gulp'));
+var request = BBPromise.promisify(require('request'));
+var util = require('gulp-util');
+
+// Set token 
+var GITHUB_ACCESS_TOKEN = process.env.GITHUB_ACCESS_TOKEN;
+var exec = BBPromise.promisify(child_process.exec);
+var gitExec = BBPromise.promisify(git.exec);
+
+var isDryrun = argv.dryrun;
+var verbose = (argv.verbose || argv.v);
+
+const issuesopt = {
+  // Url to repository 
```

Comment:
nit: Maybe remove this comment (and others like it) because they're obvious?

Reply:
done